### PR TITLE
fix: eliminate MIB dependency for ifname processor

### PR DESCRIPTION
**Pull request by Hipska**
### Required for all PRs:

- [ ] Updated associated README.md.
- [x] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

The current ifname processor required snmp MIBs to be installed while not really necessary.

### CODE REVIEW

**Review by srebhan — APPROVED** [review]
Looks good to me. Thanks @Hipska!

File: plugins/processors/ifname/ifname.go [hunk 1]
```diff
@@ -390,13 +391,13 @@ func buildMap(gs snmp.GosnmpWrapper, tab *si.Table, column string) (nameMap, err
 		if err != nil {
 			return nil, fmt.Errorf("index tag isn't a uint")
 		}
-		nameIf, ok := v.Fields[column]
+		nameIf, ok := v.Fields["ifName"]
 		if !ok {
-			return nil, fmt.Errorf("field %s is missing", column)
+			return nil, errors.New("ifName field is missing")
```
Comment (Hipska) [inline plugins/processors/ifname/ifname.go:396]: @srebhan I was wondering if this is the correct way to do it, if there are no arguments to the message.

Comment (srebhan) [reply plugins/processors/ifname/ifname.go:389]: Yes that's the correct way, using `errors.New()` if you don't have arguments.

**Review by reimda — APPROVED** [review]

**Comment by reimda** [comment]
Thanks @Hipska!

**Pull request merged**